portal-backend/depmap/data_access/breadbox_dao.py

```diff
@@ -45,7 +42,7 @@ def get_all_matrix_datasets() -> list[MatrixDataset]:
     """
     matrix_datasets = []
     for dataset in _get_breadbox_datasets_with_caching():
-        if dataset.format_ == MatrixDatasetResponseFormat.MATRIX_DATASET:
+        if dataset.format_ == "matrix_dataset":
```

Comment:
Similarly, I only changed this because the newer versions of the breadbox client return a different type for the dataset format field.